code/modules/mob/living/carbon/xenomorph/death.dm

```diff
@@ -27,8 +27,9 @@
 		update_icons()
 
 	if(!should_block_game_interaction(src)) //so xeno players don't get death messages from admin tests
-		var/datum/entity/xeno_death/death_log = SSentity_manager.tables[/datum/entity/xeno_death].make_new()
-		death_log.load_data(src, cause)
+		if(!(datum_flags & DF_VAR_EDITED))
+			var/datum/entity/xeno_death/death_log = SSentity_manager.tables[/datum/entity/xeno_death].make_new()
```

Comment:
why this syntax instead of DB_ENTITY(/datum/entity/xeno_death), btw? i know it wasn't introduced here, but

Reply:
I didn't know about it. Looking at it, DB_ENTITY() aka /datum/controller/subsystem/entity_manager/proc/select() does the same thing as this. Want this changed to that?

Reply:
ideally! it's the usual syntax for db stuff, eyes glazed over it the first time i looked

Reply:
done